2023/credentials/main.py

```diff
@@ -0,0 +1,34 @@
+import os
+from dotenv import load_dotenv
+from openai import OpenAI
+
+
+load_dotenv()  # Loads the .env file into the program so we can access the enviroment variables with getenv()
+
+SECRET_KEY = os.getenv("SECRET_KEY", "siekret")
+
+
+def main():
```

Comment:
I normally also add return type annotations in case there is no return type (so here I would write def main() -> None:). Not sure how strict we should be with this.

Reply:
Good catch!